src/routes/docs/products/network/dns/+page.markdoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: article
+title: Appwrite DNS service
+description: Learn about Appwrite's DNS service and how to configure domain records for your applications
+---
+
+Appwrite provides a dedicated DNS (Domain Name System) service through its `appwrite.zone` nameservers to help you manage domain records for your applications. This service is ideal for apex domains (root domains) that cannot use CNAME records due to DNS protocol limitations.
+
+The DNS service enables you to configure custom domains for Sites, Functions, and APIs while providing automatic SSL certificate management and high availability. Whether you need to set up subdomains or apex domains, Appwrite's DNS service offers a complete solution.
+
+# Benefits {% #benefits %}
+
+- **Support for apex domains**: Use root domains like `example.com` directly without workarounds
+- **Automatic SSL certificate management**: All domains get valid SSL certificates automatically
+- **Integrated management**: Simplified configuration for Sites, Functions, and APIs
+- **High availability**: Built on reliable infrastructure with global distribution
+
+# Managing DNS records {% #managing-dns-records %}
+
+You can manage DNS records for your domains in two places within the Appwrite Console:
+
+## Organization-level DNS management
+
+To manage DNS records at the organization level:
+
+1. Navigate to your organization in the Appwrite Console
+2. Select the **Domains** tab
+3. Here you can view and manage all your DNS records across different projects
+
+This is the central place to manage domain records when you're using Appwrite's DNS service by configuring NS records to point to `appwrite.zone`.
```

Comment:
Don't we need 2 nameservers? `ns1.appwrite.zone`, `ns2.appwrite.zone`. Not all registrars support only 1. cc @eldadfux